### DISCUSSION TITLE
Approved for student educational benefits a week ago, but haven't received it yet (or any email) - what do I do?

### DISCUSSION THREAD

**Discussion by Caeci** [community/community#119910] [GitHub Education]
### Hi everyone,

Hi everyone, the title states my problem. Does anyone know what to do?

**Comment by samus-aran** [comment]
Hi @Caeci 👋 

We believe that the application status is slightly inaccurate in your submission. We can confirm that you were only approved less than 3 days ago (April 15th) so you are still within the 72 hours of the waiting period. Did you submit the application on the 15th?

Please reach out to us if you still haven't received your academic email within the next 48 hours 👍
